docs/06-contributors/020-development.md

```diff
@@ -105,7 +105,7 @@ separately).
 
 :::note
 
-The first time you run `npm install` you may be asked to enter your system password, this is because
+The first time you run `npm install` it may take a long time, this is because
```

Comment:
How long?

Reply:
Depends on network conditions, it took a minute in my hotel room